dev/com.ibm.ws.cdi.extension.bce_fat/build.gradle

```diff
@@ -0,0 +1,11 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+apply from: '../wlp-gradle/subprojects/fat.gradle'
```

Comment:
is this file needed?

Reply:
Yes. The tasks to build and run the tests are not present without it.

Reply:
I must be missing something else in that case. I'll have a poke around and see if I can work out what it is.

Reply:
Needed `fat.project: true` instead of `test.project: true` in the bnd file.